mock-interactions.js

```diff
@@ -240,7 +240,46 @@
     makeMouseEvent('mouseup', xy, node);
   }
 
-  /**
+    /**
+     * Fires a `mouseenter` mouse event on a specific node, at a given set of coordinates.
+     * This event bubbles and is cancellable. If the (x,y) coordinates are
+     * not specified, the middle of the node will be used instead.
+     *
+     * @param {!Element} node The node to fire the event on.
+     * @param {{ x: number, y: number }=} xy Optional. The (x,y) coordinates the mouse event should be fired from.
+     */
+    function mouseenter(node, xy) {
+        xy = xy || middleOfNode(node);
+        makeMouseEvent('mouseenter', xy, node);
+    }
+
+    /**
+     * Fires a `mouseenter` mouse event on a specific node, at a given set of coordinates.
+     * This event bubbles and is cancellable. If the (x,y) coordinates are
+     * not specified, the middle of the node will be used instead.
+     *
+     * @param {!Element} node The node to fire the event on.
+     * @param {{ x: number, y: number }=} xy Optional. The (x,y) coordinates the mouse event should be fired from.
+     */
+    function mouseover(node, xy) {
+        xy = xy || middleOfNode(node);
+        makeMouseEvent('mouseover', xy, node);
+    }
+
+    /**
+     * Fires a `mouseleave` mouse event on a specific node, at a given set of coordinates.
+     * This event bubbles and is cancellable. If the (x,y) coordinates are
```

Comment:
Similarly to `mouseenter`, `mouseleave` doesn't bubble and is not cancellable.
Also, it should fire a separate event for each parent up to <body> - see https://developer.mozilla.org/en-US/docs/Web/Events/mouseleave. Do we expect this to be done here or by the user?